src/network/tls.py

```diff
@@ -88,14 +114,13 @@ def readable(self):
         except AttributeError:
             return AdvancedDispatcher.readable(self)
 
-    def handle_read(self):
+    def handle_read(self):  # pylint: disable=inconsistent-return-statements
```

Comment:
This method doesn't have to return anything, nothing checks its return value. But I probably merged two statements into one (perform appropriate I/O and skip the others). Maybe it should explicitly return something. Same with handle_write below.

Reply:
Since nothing checks return values, we might as well be consistent in not returning anything.